backtesting/backtesting.py

```diff
@@ -1078,7 +1079,7 @@ def __init__(self,
             warnings.warn('Data index is not datetime. Assuming simple periods, '
                           'but `pd.DateTimeIndex` is advised.',
                           stacklevel=2)
-
+        self._close_all_at_end = close_all_at_end
```

Comment:
Cast to `bool()` so passing `=1` is supported.

Reply:
Done.